test/jasmine/tests/download_test.js

```diff
@@ -164,6 +165,33 @@ describe('Plotly.downloadImage', function() {
         .catch(failTest)
         .then(done);
     });
+
+    it('should default width & height for downloadImage to match with the live graph', function(done) {
+        spyOn(Lib, 'isSafari').and.callFake(function() { return true; });
+        spyOn(helpers, 'octetStream');
+
+        var fig = {
+            data: [{y: [0, 1]}]
+        };
+
+        gd.style.width = '500px';
+        gd.style.height = '300px';
+
+        Plotly.plot(gd, fig)
```

Comment:
I assume this test would fail without my change?

Reply:
💃  if so!

Reply:
> I assume this test would fail without my change?

Yes it used to fail without the changes proposed by you. 
Thanks very much @nicolaskruchten for fixing this bug!